contracts/bprotocol/avatar/AbsCToken.sol

```diff
@@ -153,43 +162,47 @@ contract AbsCToken is Cushion {
         address payable userOrDelegatee
     ) external onlyBToken postPoolOp(true) returns (uint256) {
         uint256 result = cToken.borrow(borrowAmount);
+        if(result != 0) return result;
```

Comment:
it will still invoke the postpoolop, is it ok?

Reply:
don't see any problem as it will call `_hardReevaluate` and just check `canUntop()`.